src/content/docs/Company/ceo.md

```diff
@@ -29,23 +27,26 @@ Prior to Google Eric lead a small startup for 15 months as part of
 Y-Combinator's Summer 2006 batch, and prior to that worked on Safari at Apple
 where he brought SVG to Safari and helped start the Safari on Windows project.
```

Comment:
Is it worth having something like

"for more background and information on Eric please connect with him on [LinkedIn](Link goes here)"

Reply:
Sure 🤷  There is also a socials section below which we could pull up.